_includes/cookie-consent.html

```diff
@@ -56,13 +61,11 @@
             padding: 1.3rem 1.1rem;
         }
 
-        #cookie-notice-text {
-            width: 100%;
+        .buttons-block {
+            margin-top: 40px;
         }
 
-        #cookie-notice #cookie-notice-accept {
-            margin-top: 1.3rem;
-            margin-left: 0;
+        #cookie-notice-text {
             width: 100%;
         }
```

Comment:
Please add margin bottom for #cookie-notice-text instead add margin top to .buttons-block